src/theory/quantifiers/instantiate.cpp

```diff
@@ -690,7 +692,7 @@ void Instantiate::notifyEndRound()
             << " * " << i.second << " for " << i.first << std::endl;
       }
     }
-    if (options::debugInst())
+    if (outputInst)
```

Comment:
Update the use of `out` below to `Output(options::OutputTag::INST);`

Reply:
I refactored printing in the whole function since `outputInst` and `debugInstTrace` don't have anything in common.